applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/prj.conf

```diff
@@ -141,6 +138,9 @@ CONFIG_MCUMGR_TRANSPORT_WORKQUEUE_STACK_SIZE=4096
 CONFIG_MCUMGR_GRP_OS=y
 CONFIG_MCUMGR_GRP_OS_MCUMGR_PARAMS=y
 
+# Enable Zephyr System Power Management
```

Comment:
For other configurations we do not enable `CONFIG_POWEROFF`. Can we rely on imply also here, please? (to simplify configuration)